src/app/producer-shelf/components/producer-shelf/producer-shelf.component.html

```diff
@@ -1,17 +0,0 @@
-<div [style.flex-basis.px]="dragOffsetInPixels" [style.height.px]="dragOffsetInPixels">
```

Comment:
This file is now empty. Is that on purpose?

Reply:
It is, i just wanted it to exist so i could show the selector structure still. But as we discussed, the content inside it was useless, so for now i scrubbed it.